### DISCUSSION TITLE
How to remove the directory name?

### DISCUSSION THREAD

**Discussion by itsmrad** [SilentVoid13/Templater#1067] [Help]
I found this templater code from internet; How can I remove the "Daily Journal" that appears in front of the date and still create new notes in Daily Journal directory when I click on the yesterday or tomorrow link? If I remove the "Daily Journal" then it creates the new note outside the daily journal folder.

```
<%*

const currentMoment = moment(tp.file.title, "DD-MM-YYYY");

const hash = '# ';

const slash = ' / ';

const pipe = ' | ';

const leftAngle = '❮ ';

const rightAngle = ' ❯';

tR += leftAngle;

tR += '[[' + currentMoment.format('YYYY') + ']]' + slash;

tR += '[[' + currentMoment.format('YYYY-[Q]Q|[Q]Q') + ']]' + slash;

tR += '[[' + currentMoment.format('YYYY-MM|MMMM') + ']]' + slash;

tR += '[[' + currentMoment.format('gggg-[W]ww') + '|' + currentMoment.format('[Week] ww') + ']]';

tR += rightAngle;

tR += '\n';

tR += leftAngle;

currentMoment.add(-1,'days');

tR += '[[Daily Journal/' + currentMoment.format('DD-MM-YYYY') + ']]' + pipe;

currentMoment.add(1,'days');

tR += currentMoment.format('DD-MM-YYYY') + pipe;

currentMoment.add(1,'days');

tR += '[[Daily Journal/' + currentMoment.format('DD-MM-YYYY') + ']]';

currentMoment.add(-1,'days');

tR += rightAngle;

%>
```

**Comment by HyperEpsilon** [comment]
The daily journal folder path needs to be there in order to create the file in the correct place, however you could hide it using an alias if you want.

`tR += '[[Daily Journal/' + currentMoment.format('DD-MM-YYYY') + pipe + currentMoment.format('DD-MM-YYYY') ']]';`
`[[Daily Journal/30-03-2023|30-03-2023]]`

This should produce the result you want. It won't update the display name if you ever change the filename, but seeing as it's a daily note I don't really see that happening
